news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+
+### ::{ flag=KR }:: South Korea vs. ::{ flag=US }:: United States
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116363241)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2315417594)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/-H-rOKG-2C4?start=645&end=948" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Throughout this world cup, and throughout the finals weekend, there has been one matchup on people's minds — the anticipated Grand Finals matchup, and the matchup from Grand Finals in 2023, South Korea vs. the United States. With how close last year's Grand Finals was and how much both teams improved in the year since, expectations for a close, high-level match were sky-high, and this winners' Finals delivered.
+
+The match started off relatively predictably — South Korea banned out [Bound to Madness](https://osu.ppy.sh/beatmapsets/2284575#osu/4872589) and [Blue Night](https://osu.ppy.sh/beatmapsets/31333#osu/116006), the two most speed-focused maps in the pool, and the United States banned out [Dynasty of Extinction](https://osu.ppy.sh/beatmapsets/2284670#osu/4872796) and [Lunatic Eyes](https://osu.ppy.sh/beatmapsets/2267484#osu/4872667), both in the hidden pool. However, expectations would soon be flipped as the United States took South Korea's first pick, [Ratpark](https://osu.ppy.sh/beatmapsets/2284654#osu/4872765), by only 87,222 points.
+
+This gave the United States a solid lead through the next six picks, until finally, on the United States' fourth pick and the eighth map overall, [Dear the Night I Loved](https://osu.ppy.sh/beatmapsets/2284389#osu/4872029), South Korea finally took back the early break point, tying up the match 4–4. The teams continued trading picks back and forth until finally, with South Korea having first pick, the score was 6–5 for South Korea.
+
+Throughout the match, there were displays of massive skill on a wide variety of maps and margins of difference ranging from 87k to 1.6 million points. These teams proved to be very evenly matched throughout this bout, and with the United States having last pick, the match seemed almost destined to go to tiebreaker — until suddenly it didn't.
+
+The United States last picked, [Horobi o Inorishi Mono](https://osu.ppy.sh/beatmapsets/2284565#osu/4872576), the Hard Rock precision, low bpm stream pick. While [Rektygon](https://osu.ppy.sh/users/7813296) would go on to top score the map with a pop-off 627,186 points, the second-highest score of the weekend, it would ultimately not be enough. South Korea took this last pick by a mere 84,436 points, closing out the match 7–5 and securing their spot in the top two and on the winners' side of the grand finals.
```

Comment:
```suggestion
The United States picked [Horobi o Inorishi Mono](https://osu.ppy.sh/beatmapsets/2284565#osu/4872576), the Hard Rock precision low-BPM stream pick. While [Rektygon](https://osu.ppy.sh/users/7813296) would go on to top-score the map with a pop-off 627,186 points, the second-highest score of the weekend, it would ultimately not be enough. South Korea took this last pick by a mere 84,436 points, closing out the match 7–5 and securing their spot in the top two and on the winners' side of the Grand Finals.
```